src/gseproject/EXEplorer/urls.py

```diff
@@ -16,4 +17,6 @@
     path('api/get_carbon_footprint_value/', views.get_carbon_footprint_value, name='get_carbon_footprint_value'),
     path('api/increment_carbon_footprint/', views.increment_carbon_footprint, name='increment_carbon_footprint'),
     path('scan/', views.scan_view, name='scan'),
+    path('start-pet-walk/', views.petDisappear, name='start-pet-walk'),
+    path('pet-check', views.petCheck, name = 'pet-check'),
```

Comment:
Do these need their own urls? can the attached functions be triggered without a url

Reply:
The attached functions cannot be triggered without a url. Its also API